warehouse/query-core/src/test/java/datawave/query/jexl/visitors/ExecutableExpansionVisitorTest.java

```diff
@@ -134,14 +136,16 @@ protected void runTestQuery(List<String> expected, String querystr, Date startDa
     
     @Deployment
     public static JavaArchive createDeployment() throws Exception {
+        System.setProperty("cdi.bean.context", "queryBeanRefContext.xml");
```

Comment:
Not super critical, but I don't know this stuff too well (and maybe other devs won't). Perhaps add a comment explaining why we need to add this property/file? Also, consider adding comments why we add/delete certain packages and classes in the call to `ShrinkWrap` below

Reply:
I wish I knew.  I copied this from a different class to get it working.  I will try to get my head around it however.

Reply:
added what I could figure out.  The MockAlternative addition is still stumping me because nothing is actually marked with that prototype.